pkg/pillar/cmd/nim/nim.go

```diff
@@ -791,7 +790,26 @@ func (n *nim) ingestDevicePortConfigFile(oldDirname string, newDirname string, n
 		return
 	}
 	dpc.DoSanitize(n.Log, true, false, "", true, true)
-	dpc.OriginFile = filename
+	// Compare file timestamp first against /run/zedagent/DevicePortConfig/zedagent.json
+	// (if it exists and is newer we ignore the file from /config), and then
+	// /persist/status/nim/DevicePortConfigList/global.json
+	// and . If the latter exists and is newer
```

Comment:
Probably a mistake in the comment - "and ." can be removed.

Reply:
Fixed